# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/x11/xfree86/gcc34.patch.gatos
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -dur ati.4.4.0/fi1236.h ../src.xfree86.1097053505.16606.2407344815/xc/gatos-ati.4.4.0-2004-08-09/ati.4.4.0/fi1236.h
--- ati.4.4.0/fi1236.h	2004-07-14 07:38:24.000000000 +0200
+++ ../src.xfree86.1097053505.16606.2407344815/xc/gatos-ati.4.4.0-2004-08-09/ati.4.4.0/fi1236.h	2004-10-06 12:42:16.588955600 +0200
@@ -89,6 +89,7 @@
 int FI1236_AFC(FI1236Ptr f);
 int TUNER_get_afc_hint(FI1236Ptr f);
 void fi1236_dump_status(FI1236Ptr f);
+static void MT2032_dump_status(FI1236Ptr f);
 
 #define FI1236SymbolsList  \
 		"Detect_FI1236", \
diff -dur ati.4.4.0/theatre.h ../src.xfree86.1097053505.16606.2407344815/xc/gatos-ati.4.4.0-2004-08-09/ati.4.4.0/theatre.h
--- ati.4.4.0/theatre.h	2004-05-20 03:37:59.000000000 +0200
+++ ../src.xfree86.1097053505.16606.2407344815/xc/gatos-ati.4.4.0-2004-08-09/ati.4.4.0/theatre.h	2004-10-06 12:45:01.497885632 +0200
@@ -53,6 +53,7 @@
 void RageTheatreDebugGain(TheatrePtr t, Bool on, CARD32 gain);
 void ShutdownTheatre(TheatrePtr t);
 void DumpRageTheatreRegs(TheatrePtr t);
+void DumpRageTheatreRegsByName(TheatrePtr t);
 void ResetTheatreRegsForTVout(TheatrePtr t);
 
 
